workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/RequiredFieldInjector.cs

```diff
@@ -27,6 +27,8 @@ public class RequiredFieldInjector
             = "[Require] attribute found on member that is not Injectable. This member will be ignored.";
         private const string MalformedInjectable
             = "Injectable found without required attributes, this is invalid.";
+        private const string RequirableFieldDoesNotInheritRequirableBase
+            = "[Require] field element does not inherit RequirableBase. This is most likely a bug in the SpatialOS GDK.";
```

Comment:
I don't think user should be exposed to RequirableBase. It doesn't really mean anything to them.